cf-testd/cf-testd.c

```diff
@@ -458,8 +512,27 @@ int CFTestD_StartServer()
     return 0;
 }
 
+static void HandleSignal(int signum)
+{
+    switch (signum)
+    {
+    case SIGTERM:
+    case SIGINT:
+        // flush all logging before process ends.
+        fflush(stdout);
+        exit(EXIT_FAILURE);
+        break;
+    default: break;
+    }
+
+    signal(signum, HandleSignal);
```

Comment:
Why this call?

Reply:
I don't have a ton of experience with signal handling. I thought I needed to reset the signal handler. Of course it is a no-op with the current code since I only register SIGINT and SIGTERM and both of those cause an exit. I will remove.